app/source/main.cpp

```diff
@@ -263,17 +64,16 @@ int main()
 	// Initialize the scene
 	sceneInit();
 
-	// get the local account id for the current enviroment and set the selected button to it
-	FRDA_GetLocalAccountId(&currentEnv);
-	environment = AccountToNascEnvironment(currentEnv);
+	// set button selected and current account to nasc environment
+	u32 serverTypes[3] = {};
+	FRDU_GetServerTypes(serverTypes);
 	
-	if (currentEnv == Account::Undefined) currentEnv = Account::Nintendo;
-	
-	buttonSelected = currentEnv;
+	mainStruct.buttonSelected = static_cast<NascEnvironment>(serverTypes[0]);
+	mainStruct.currentAccount = mainStruct.buttonSelected;
 
 	// Main loop
 	while (aptMainLoop()) {
-		bool exit = false;
+		u32 exit = 0;
```

Comment:
Why is exit a u32 now? All uses of it return a bool

Reply:
Im gonna be real with you, i have no idea why i made that change, probably eventual error stuff or something

Reply:
Well reverting it back shouldn't break anything I think. In fact I think this is currently broken because we are checking below if exit (a u32) is below zero

Reply:
reverted to bool